packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/AutocompleteCreatedBy/AutocompleteCreatedBy.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+
+import { components } from '@/api';
+import { useMoniteContext } from '@/core/context/MoniteContext';
+import { MoniteScopedProviders } from '@/core/context/MoniteScopedProviders';
+import { useRootElements } from '@/core/context/RootElementsProvider';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Autocomplete, CircularProgress, TextField } from '@mui/material';
+
+interface AutocompleteCreatedByProps {
+  onChange: (id?: string) => void;
+}
+
+type EntityUserType = components['schemas']['EntityUserResponse'];
+
+export const AutocompleteCreatedBy = (props: AutocompleteCreatedByProps) => (
+  <MoniteScopedProviders>
```

Comment:
You don't have to wrap the inner component with `<MoniteScopedProviders>`. Only publicly exported components must be wrapped.

Reply:
removed